conn.go

```diff
@@ -549,39 +555,50 @@ var eventWatchTypes = map[EventType][]watchType{
 	EventNodeDataChanged:     {watchTypeExist, watchTypeData, watchTypePersistent, watchTypePersistentRecursive},
 	EventNodeChildrenChanged: {watchTypeChild, watchTypePersistent},
 	EventNodeDeleted:         {watchTypeExist, watchTypeData, watchTypeChild, watchTypePersistent, watchTypePersistentRecursive},
+	EventPingReceived:        nil,
```

Comment:
Nit: might be better to have an empty slice? Avoid dealing with `nil`s...

Reply:
Actually, it's definitely preferred in go to use a nil slice here instead of an empty one. They are functionally identical in almost all cases (see https://gosamples.dev/empty-vs-nil-slice/).

Reply:
TIL, thanks for sharing.